docs/UserGuide.md

```diff
@@ -3,271 +3,432 @@ layout: page
 
-## Quick start
+* [Why ClientBook?](#why-clientbook)
+* [Quick Start](#quick-start)
+* [Here's what we can offer you!](#heres-what-we-can-offer-you)
+* [Feature Description](#feature-description)
+  * [Viewing help : `help`](#viewing-help--help)
+  * [Add client contact: `add`](#add-client-contact-add)
+  * [Edit client contact: `edit`](#edit-client-contact-edit)
+  * [List all clients : `list`](#list-all-clients--list)
+  * [Search for client contact based on keywords: `find`](#search-for-client-contact-based-on-keywords-find)
+  * [Display policies associated with selected client: `policy`](#display-policies-associated-with-selected-client-policy)
+  * [Delete client: `delete`](#delete-client-delete)
+  * [Sort list of clients: `sort`](#sort-list-of-clients-sort)
+  * [Lock ClientBook with a user-selected password: `lock`](#lock-clientbook-with-a-user-selected-password-lock)
+  * [Unlock ClientBook : `unlock`](#unlock-clientbook--unlock)
+  * [Exiting the program : `exit`](#exiting-the-program--exit)
+  * [Saving data](#saving-data)
+  * [Editing data file](#editing-data-file)
+  * [Upcoming features!](#upcoming-features-v20-and-beyond)
+* [Frequently Asked Questions](#frequently-asked-questions)
+  * [Setting Up](#setting-up)
+* [Command Summary](#command-summary)
+* [Contact Us](#contact-us)
 
-1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `clientbook.jar`. (A release will be coming out soon)
+## Why ClientBook?
+
+ClientBook is designed for insurance agents who are always on the go. We understand the **pain** and **inconvenience** of having to **navigate around 
+your screen without a mouse** while you are out meeting a client, as well as the **limitations of a laptop's battery life**.<br>
+
+Thus, ClientBook is born!<br>
+
+Requiring **minimal interactions with your cursor**, ClientBook still allows you to complete essential contact management 
+tasks with our minimal yet essential set of contact management features.<br> 
+
+Consisting of only the **most necessary features** any good insurance agent will need while on the go, ClientBook is hence extremely easy on your laptop's battery!
+
+
+
+## Quick Start
+
+1. Ensure you have Java `11` or above installed in your Computer.
+   * To check if you have this installed or for instructions on how to install, refer to our FAQ [here](#FAQ).
 
+1. Download the latest `clientbook.jar` [here](https://github.com/AY2021S2-CS2103T-W15-2/tp/releases/tag/v1.2).
 1. Copy the file to the folder you want to use as the _home folder_ for your ClientBook.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+1. Double-click the jar file to start the app. The GUI similar to the below should appear in a few seconds. 
```

Comment:
Instead of "jar file" maybe you can put "clientbook.jar" with a markup to be more specific?

Reply:
Okay, changed!